bootloader/src/pyi_archive.c

```diff
@@ -377,6 +378,145 @@ findDigitalSignature(ARCHIVE_STATUS * const status)
 #endif /* ifdef _WIN32 */
 }
 
+/*
+ * Try to find the pydata section in an ELF executable.
+ * Returns the end of the section if found.
+ */
+static int
+findElfSection(ARCHIVE_STATUS * const status)
+{
+#if !defined(WIN32) && !defined(__APPLE__)
```

Comment:
This seems unnecessary as there is already a platform check macro around the function call.

Reply:
The conditionals here mirror the respective conditionals in `findDigitalSignature`.

@htgoebel I'll remove those and check if it compiles on windows and apple.